src/main/resources/resources/logisim/strings/std/std.properties

```diff
@@ -858,6 +858,7 @@ pinPullAttr = Pull Behavior
 pinPullDownOption = Pull Down
 pinPullNoneOption = Unchanged
 pinPullUpOption = Pull Up
+pinInitialValue = Reset value:
```

Comment:
The key is `initialValue` but the value is `Reset value`. Can we have this more consistent - would be less confusing for translators.

Reply:
Won´t fix, Initial or Reset is for me the same

Reply:
I concur with @MarcinOrlowski in this matter. I think "initial value" is more appropriate. Also, there's a semantical difference in the context of discrete event simulators: Initial value signifies the initial value of the signal at start of simulation after end of elaboration. The reset value is the value imposed if a corresponding reset signal becomes active and imposes the change of the signal value.

Reply:
My point is that by work meaning `initial value` and `reset value` are not equivalent. You can start with `initial value` and then reset to `reset value` and these two do not need to be the same. Also for me `reset value` does not automatically implies that this is also the `initial value`. Can we therefore meet in-between and set value to `Initial/reset value`? That would **explicitly** cover both cases and left no room for misinterpretation. 

@davidhutchens @maehne @BFH-ktt1

Reply:
@MarcinOrlowski : I agree with your discussion, would, however, not take `initial value` as this is a programming term. Real hardware does not have an `initial value`, but is set to a defined state by a reset. Hence, therefore I would opt to stay with `reset value`. What do you think?

Reply:
@BFH-ktt1 I am buying the "real hardware" argument as justification for that wording. Let's move on. Thanks.